release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -60,6 +60,72 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-6-rhcos"]
 === {op-system-first}
 
+[id="ocp-4-6-boot-live-iso-by-running-coreos-installer"]
+==== {op-system} PXE and ISO now live environment
+
+The PXE media and ISO available for {op-system} are now a fully live
+environment. Unlike the previous dedicated PXE media and ISO used for
+installation, the {product-title} live environment can be configured via
+Ignition and contains all the same packages as the main {op-system}, such as
+podman. This allows arbitrary scripting of pre/post installation workflows. For
+example, you could run `coreos-installer` and then make an HTTP request to
+signal success to a provisioning server. PXE boots use the normal
+`ignition.config.url`. The ISO can be configured via Ignition by using the
+following command:
+
+[source,terminal]
+----
+$ coreos-installer iso ignition embed
+----
+
+[id="ocp-4-6-refactored-coreos-installer"]
+==== Refactored `coreos-installer`
+
+The `coreos-installer` has been rewritten to support more features, some of
+which include:
+
+* Modifying the kernel arguments of the installed system.
+* Fetching Ignition configs.
+* Saving previously existing partitions.
+* Using the `coreos-installer iso ignition` command to configure Ignition for
+the new live ISO.
```

Comment:
"Configuring Ignition for the new live ISO using the `coreos-installer iso ignition` command"?